package.json

```diff
@@ -116,6 +116,7 @@
     "node": ">= 6.0.0",
     "npm": ">= 3.0.0"
   },
+
```

Comment:
Please remove this whitespace

Reply:
whitespace removed

Reply:
Apologies, I know I asked you to remove this whitespace, and I'm not sure what I was thinking when I asked. Please add the whitespace after the colon back :/